lms/templates/vert_module.html

```diff
@@ -69,6 +69,10 @@ <h2 class="hd hd-2 unit-title">${unit_title}</h2>
 % endfor
 </div>
 
+% if reset_button:
+  <button type="button" class="problem-reset-btn" data-value="${_('Reset Problems')}"><span aria-hidden="true">${_('Reset Problems')}</span><span class="sr">${_("Reset Problems")}</span></button>
```

Comment:
Let's add a `btn-link` class here. I believe it looks more consistent with the current interface.